Source/SwiftLintFramework/Extensions/SwiftExpressionCallKind.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SwiftExpressionCallKind.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 11/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public enum SwiftExpressionCallKind: String {
```

Comment:
It's fine to keep `public`, especially since SwiftLint is pre-1.0 (_still_ 😅). But this really _does_ belong in SourceKittenFramework (as a `public` API, of course), and we certainly _should_ move it there before SwiftLint gets to 1.0 with a CLI and framework interface we want to maintain following SemVer.

To be clear, you're just opposed to making this `internal` at the moment, right? We can still rename to `SwiftExpressionKind` and add the other members?